velox/exec/MergeJoin.h

```diff
@@ -331,6 +331,10 @@ class MergeJoin : public Operator {
   // rows from the left side that have a match on the right.
   RowVectorPtr filterOutputForAntiJoin(const RowVectorPtr& output);
 
+  void clearLeftInput();
```

Comment:
nit: Please add some comments for `clearLeftInput ` and `clearRightInput ` method.